include/EffectLabelButton.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * EffectLabelButton.h - class trackLabelButton
+ *
+ * Copyright (c) 2024 Noah Brecht <noahb2713/at/gmail/dot/com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#ifndef LMMS_GUI_EFFECT_LABEL_BUTTON_H
+#define LMMS_GUI_EFFECT_LABEL_BUTTON_H
+
+#include <QPushButton>
+
+namespace lmms::gui
+{
+
+class EffectView;
+
+class EffectLabelButton : public QPushButton
```

Comment:
I have just noticed that this class inherits from `QPushButton` whereas `TrackLabelButton` inherits from `QToolButton`. I think the letter is done so that the button can be put into a checked state when for example an instrument window is opened. So I wonder if the same would make sense for opened effects.